op-e2e/system/da/da_throttling_test.go

```diff
@@ -72,10 +71,14 @@ func TestDATxThrottling(t *testing.T) {
 	// second tx should still be throttled
 	require.Nil(t, bigReceipt, "large tx did not get throttled")
 
-	// disable throttling to let big tx through
-	batcher.Config.ThrottleParams.TxSizeUpperLimit = math.MaxUint64
-	batcher.Config.ThrottleParams.TxSizeLowerLimit = math.MaxUint64 - 1
-	err = batcher.SetThrottleController(config.StepControllerType, nil) // We need to set the controller again to propagate the change
+	// Try setting the disabled controller, this is expected to return an error
+	// in the current implementation.
+	err = batcher.SetThrottleController(config.DisabledControllerType, nil)
+	require.Error(t, err)
```

Comment:
why is it expected to return an error?

Reply:
Ah that makes sense. Yes agree, it's an edge case to support runtime dis- and enabling. It may be worth documenting that it's not possible to disable (or enable) during runtime and that the batcher has to be restarted for this.